config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "typeScale": [8.4, 6.4, 4, 3.2, 2.4, 1.8, 1.6, 1.4, 1.2, 1],
+  "typeScale": [9.6, 6, 4.8, 3.4, 2.4, 2, 1.6, 1.4, 1.2, 1],
```

Comment:
You'll wanna be careful with this bc it'll certainly break all `f(something)`

Reply:
Yep, my rational on this is that the changes are very small in size, and we use don't use this sizes a lot through out the code.
Also, since we are focusing on building the new app, I don't think we need to care a lot for deprecated pages